src/main/java/io/fabric8/maven/docker/access/hc/DockerAccessWithHcClient.java

```diff
@@ -413,7 +413,8 @@ public boolean removeImage(String image, boolean... forceOpt) throws DockerAcces
 
             return response.getStatusCode() == HTTP_OK;
         } catch (IOException e) {
-            throw new DockerAccessException(e, "Unable to remove image [%s]", image);
+            log.warn( "Unable to remove image [%s]", image);
```

Comment:
Question here is, why an `IOException` is thrown, since when an Image is not found, it shouldn't throw an exception but return a 404 (which is allowed in line 409 without throwing an exception). 

In any other case, i.e. when the call itself went wrong and an IOException occurs, I probably wouldn't want to continue as usual.

Do you have a specific scenario and the specific IOException that occurs for you ?